prettify-plugin/src/js/config.js

```diff
@@ -0,0 +1,79 @@
+jQuery.noConflict();
+
+(function($, PLUGIN_ID) {
+    'use strict';
+    const CONF = kintone.plugin.app.getConfig(PLUGIN_ID);
+
+    const $form = $('.js-submit-settings');
+    const $cancelButton = $('.js-cancel-button');
+    const $text = [];
+
+    function setDropdown() {
+        return KintoneConfigHelper.getFields('MULTI_LINE_TEXT').then(function(resp) {
+            for (let i = 0; i < resp.length; i++) {
```

Comment:
It seems as if you can merge the `for` loop at line 13 and the `forEach` loop at line 32.